resources/tap.rb

```diff
@@ -19,13 +19,49 @@
 # limitations under the License.
 #
 
-actions :tap, :untap
-default_action :tap
-
-attribute :name,
-          name_attribute: true,
-          kind_of: String,
+property :name,
+          name_property: true,
+          String,
           regex: %r{^[\w-]+(?:\/[\w-]+)+$}
 
-attribute :tapped,
-          kind_of: [TrueClass, FalseClass]
+property :tapped,
+          [true, false]
+
+
+          include ::Homebrew::Mixin
```

Comment:
Was thinking more the horizontal spacing with lines 31 - 67 being over 10 spaces.